docs/documentation/development/bindings/thing-handler.md

```diff
@@ -180,6 +180,27 @@ Furthermore bindings can specify a localized description of the thing status by
 rate_limit=Device is blocked by remote service for {0} minutes. Maximum limit of {1} configuration changes per {2} has been exceeded. For further info please refer to device vendor.
 ```
 
+## Offering a callback URL
```

Comment:
I am not sure whether this is the correct place for the documentation of this feature as the feature is completely independent of bindings. It is often used for the implementation of AudioSinks and potentially other extensions. We should rather have a place for such general system services.